in_toto/in_toto_run.py

```diff
@@ -40,106 +74,47 @@
-            elements are arguments passed to that command.
-    record_streams:
-            A bool that specifies whether to redirect standard output and
-            and standard error to a temporary file which is returned to the
-            caller (True) or not (False).
-    signing_key:
-            If not None, link metadata is signed with this key.
-            Format is securesystemslib.formats.KEY_SCHEMA
-    gpg_keyid:
-            If not None, link metadata is signed with a gpg key identified
-            by the passed keyid.
-    gpg_use_default:
-            If True, link metadata is signed with default gpg key.
-    gpg_home:
-            Path to GPG keyring (if not set the default keyring is used).
-
-  <Exceptions>
-    SystemExit if any exception occurs
-
-  <Side Effects>
-    Calls sys.exit(1) if an exception is raised
-
-  <Returns>
-    None.
-  """
-
-  try:
-    runlib.in_toto_run(step_name, material_list, product_list, link_cmd_args,
-         record_streams, signing_key, gpg_keyid, gpg_use_default, gpg_home)
-
-  except Exception as e:
-    log.error("(in-toto-run) - {0}: {1}".format(type(e).__name__, e))
-    sys.exit(1)
-
 def main():
   """Parse arguments, load key from disk (prompts for password if key is
   encrypted) and call in_toto_run. """
 
   parser = argparse.ArgumentParser(
       description="Executes link command and records metadata")
-  # Whitespace padding to align with program name
-  lpad = (len(parser.prog) + 1) * " "
-
-  parser.usage = ("\n"
-      "%(prog)s  --step-name <unique step name>\n{0}"
-               "{{--key <functionary signing key path>, "
-               " --gpg [<functionary gpg signing key id>]}} \n{0}"
-               "[--gpg-home <path to gpg keyring>]\n{0}"
-               "[--materials <filepath>[ <filepath> ...]]\n{0}"
-               "[--products <filepath>[ <filepath> ...]]\n{0}"
-               "[--record-streams]\n{0}"
-               "[--no-command]\n{0}"
-               "[--verbose | --quiet] -- <cmd> [args]\n\n"
-               .format(lpad))
-
-  in_toto_args = parser.add_argument_group("in-toto options")
+
+  parser.usage = ("%(prog)s <named arguments> [optional arguments]"
+      " -- <command> [args]")
+
+  named_args = parser.add_argument_group("required named arguments")
 
   # FIXME: Do we limit the allowed characters for the name?
-  in_toto_args.add_argument("-n", "--step-name", type=str, required=True,
-      help="Unique name for link metadata")
+  named_args.add_argument("-n", "--step-name", type=str, required=True,
+      help="Unique name for link metadata", metavar="<unique step name>")
 
-  in_toto_args.add_argument("-m", "--materials", type=str, required=False,
-      nargs='+', help="Files to record before link command execution")
+  parser.add_argument("-m", "--materials", type=str, required=False,
+      nargs='+', help="Files to record before link command execution",
+      metavar="<material path>")
```

Comment:
Should you instead use `<material paths>` (note: plural) to indicate that more than one path can be given?

Reply:
No, given that metavar, argparse will create an accurate help message: 
```
-m <material path> [<material path> ...], --materials <material path> [<material path> ...]
```